examples/jupyter/expansion-filters.ipynb

```diff
@@ -437,6 +439,1116 @@
+       "      <td>5</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P5</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>0.000111</td>\n",
+       "      <td>0.000149</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>6</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P6</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>-0.000692</td>\n",
+       "      <td>0.000177</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>7</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P7</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>-0.000064</td>\n",
+       "      <td>0.000152</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>8</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P8</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>-0.000139</td>\n",
+       "      <td>0.000142</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>9</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P9</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>0.000201</td>\n",
+       "      <td>0.000119</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>10</td>\n",
+       "      <td>4</td>\n",
+       "      <td>P10</td>\n",
+       "      <td>total</td>\n",
+       "      <td>flux</td>\n",
+       "      <td>-0.000051</td>\n",
+       "      <td>0.000112</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    cell spatiallegendre nuclide score      mean  std. dev.\n",
+       "0      4              P0   total  flux  5.43e-01   5.99e-04\n",
+       "1      4              P1   total  flux  6.35e-04   5.23e-04\n",
+       "2      4              P2   total  flux -5.60e-02   3.42e-04\n",
+       "3      4              P3   total  flux -5.27e-05   2.84e-04\n",
+       "4      4              P4   total  flux -7.49e-04   2.30e-04\n",
+       "5      4              P5   total  flux  1.11e-04   1.49e-04\n",
+       "6      4              P6   total  flux -6.92e-04   1.77e-04\n",
+       "7      4              P7   total  flux -6.41e-05   1.52e-04\n",
+       "8      4              P8   total  flux -1.39e-04   1.42e-04\n",
+       "9      4              P9   total  flux  2.01e-04   1.19e-04\n",
+       "10     4             P10   total  flux -5.05e-05   1.12e-04"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the scaling factors for expansion coefficients will be given in Python API, thus, we do not need to multiply the moments by scaling factors."
```

Comment:
```suggestion
    "Since the scaling factors for expansion coefficients will be provided by the Python API, we do not need to multiply the moments by scaling factors."
```